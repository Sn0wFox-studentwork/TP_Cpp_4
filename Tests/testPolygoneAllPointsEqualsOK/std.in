PC pc1 2 2 2 2 2 2 2 2 2 2
LIST
EXIT
