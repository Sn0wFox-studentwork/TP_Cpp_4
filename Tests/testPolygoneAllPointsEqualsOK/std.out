OK
PC pc1 2 2 2 2 2 2 2 2 2 2